doc/source/notebooks/advanced/multioutput.ipynb

```diff
@@ -0,0 +1,747 @@
FoCMRsMBNkDMJsNH9mDvWYy9U1BrwDdDg7aOlvGZ4h4fT6qbU56HP381HzX+ilDr/HgrP2Qfr9IaTE9dAV9hRazxIM8aK9TFOPk9/eMpOV08XchvNt9oLks0ihOJcY2nwcndBxaKSC29UDxgBE7KIaNv0cDIGw5BHorhU9otIvhR2/F/UZlkSxsV0InWh1HTwOcPcJV5y7Hzz9QhzQhQBEZY0kDID4e1pT4MBfxN8rPEUkERz4E1z6iDh2qChS1yEqG1IvEWNwP/ov+PIfTmvJNqeNRnsjve7e4Q6tH9d9TKolldyI3OHjKrorqO+r5/K0y1FQMBlMmA1mAg2Bw1XOOjounwuH18GHtR+O+Ywuh4ddTSXU9x+h3lGO3dtNhG8pjdVXoOsKqqrTaovA5LydhdMGuD3va6RFhfDz62cM92EaQrbUOL+RQnEu0l4qKrkv/zHUfyo2LEMA5N8oWpbrg3fLRa+B7bCo4o5Ih82/FnfNKfPBYYN3Hzn/puj53LD5fwYtq2whEJpPbOydVTDt2rHHD9WkLHpoRAA++V9AFyJR+ZF4PW2h+Ftv/Ll4PpRMkH8jFP5K1LQ4u6HgVhEnqtkCITHQWS0E5BTEuNPZSU1vDf3e/s+81+Hs4IPaD7i74G5yI3Kp6K7guaLnmB0nXGI6Oh6/B4/fc1xR6XB2sKF+A1+f9nWClRSK2g/xccuf0dExKEbC+++iriGLtBiNqDCdfdUqGjpXpC/kjhmpTE+0Yg0yndY/j+TcRArFuUjTPrHB67qwHqbfCBXvisFIRrNwo2g+IRJmq0iZjUgTm+G2xyF7pdgAl/w7WJLA5xm0SM4xdF1YB85u8bP7edF915IIu54R1wvCfdRWImI7R1sQcdNh2vVCRKdcLSwH9BHLIW76iJBc+jBs+oWYG6IY4LIfCovN7xapyemXinhRX7MQFhDnXvow7H1JVNcv+8Hw8teVrCM/Kp+58XOp6qmiqb+Jiu4KNtRtYEXaijEbfVxwHD7dx3NFz5FqSaWmtwaDamBO3BxgrHWRaknlhZIXWJO/htyIXP5a/lf2te/j7hl3UxA9iz+UrCM6MJnGgSOATowpG1vNl6nvj2DxVD+x4Trv7jERYFDQ0dlU3s5t81KlSFzAyDbj5yqjZ1PE5cPGn8CeF2HWnWLzqv0Ekdmui9/ZK2DGLWIjLH1d3C0X3CI+S1EhNA6sySLd9ujmbKPbo4/+/vF2W2macAs5u0XMxdkt7t6HGG0N5N8kmiz6vUI0Z31VxBzaSqHw15C3WhQxthTBkQ/BFCRExxgkhkZpPvC5hPDq2mA33xDo7xCfDyJW5OoBNUB8dtXHQnRK/i7moasGITgwIjYp84Vby5rMrvb9PLz5Ye4tuJfksGSxobft48qMK/mo7iPunHYnrY5W9rXto8XRQmqYEIihjKR5cfNYkrKEPncffy3/67AwgBCOF0peYHHSYgobC0GHm7LvpKG3ix3t7+PWBlAxkKbeQOmh2QQHKlw5x49fg3d2m1BQefDyHNKjgvn3vxcB8MxX50j30nmGnEdxvnP05v3Jb0RBXt02aD4A6YuhaQ9E5Yh4ht8nNi5FhZxVULNZbGqaf6w7xhgoRCM0HoIiRPD76IFJ4zVAyesUguDqEb/dfWLTPhFDYqH5xDWjiRGzcdPFONq+RuhrYXizB4YF1RgkrDFrkrAK4meIv4Gmib9te6k43BQCnqNSj03BwtVkb4HoXGHBGAKElVL50Yhra/grDazt2EGb7uH9+g1iQ28oxK/5Sbem0+/tp8XRMny4qqgYVSMev+eYl202mIkJjiE8MJwIcwQBagBVPVXU9NUQFxzHgNeF3ds7fHyEKZm2uoW4e2eTEaexapaf4ACFw7VR9DtNrJwWx63zUgkwqmyvsvHPg82kRYXI+ojzDCkUFyI1W+Avtwg/ee5qsWlue1wIQuNe6CgTx5lChPultWjkTvtYqEao+ACS50FgGLz1gJiJsXOtsEiu/b8zs25NA69DiF9kJsTkwd4XRVwFRmILQ3EG+GwAuq1UpAfHToHif4xs6qMJi4dAq7iuIddcWKJwCWWvhIYdojlj7VZhmY3+u+x6TsQoLn1YWGxDc0NATCQMDIPuauiqGRSzIWsOyFkNwZEj2WidVfTnruKdI+t5oreIvIhcdncfIsOaQW1vLTo6oaZQBrwDaGisSF3BtKhpPFf0HCiwNGkphY2FeP1eNDTyIvKIDoqm291Nt6ubHncPbp8bDQ1VUdF1nVhzCg6fnX5fNwnqEtprr6R3AIJjN3BVfgqpITmkhORiCQwnJzaMlMjgM/JPK5ncyMFFFyJN++CO10SvKFuFeG3RQ2JzNQaIzbBuG5gtQiQUFUr+IVwqmctGPmfXc+L3/LvF5vruw0JswhJgy/+IjKqUi0V/KVOQKAA0BIgNWFEBRbivdE38aH5xp+73Cn++zzWYluoam4FkCoL3BzvkRqTDJ4+Jz7r04RHRG4o7bHsc5t0tHtdth8Zd4vGxLI6kecKqyl4pYgjTrheBfmsK9DZA3AyY9w2xrtpPRKxB84/9jNC4USLxp5EEgfJ3RHHjrDshdQFs/S1Muw66a6DuU3FdR94XLr3iPlCNOBd/m9q6Qm4o/Yjq3Pn8pfsQZtVETW8N1gALmdYs9nfsx6SaWJW6iq1NW2l1tIICd88QwexgUzDrK9eTHJpMU38Tq9JXjXE7PVf8HLoGK5Nu51CDSq3rZfx4CXFcSUXDQgKMfvJTFabmpNDjbic97jqigq3kJ1uxmGUsQjIWaVGczzi74eMfi9gDiM01YabYDC2Jwt+u+8Xmak0Rd9CBVvj0CYY36LjpYjPc/8cRIVCNI++dLmVviUAwjASAP/lfMcypt1GsY+o1Io4w+2tirR3lQugGRjWms6ZCeLKwnDQ/zLx9MNPrf4Q4xc+AjgoRlyl7Q1gBlR+L+E5biejQ27xPuOxaDnzWVTRE4a/EZ422NsrfgdZi4eI6uoaltUiMutV1GE
```

Comment:
It'd be good to point out that while the inducing points are independent, there needs to be the same *number* of inducing points per dimension.